packages/npm/@amazeelabs/recipes/recipes/add-storybook.ts.md

```diff
@@ -0,0 +1,243 @@
+# Add storybook UI and Tailwind
+
+This recipe will create the necessary structure and configuration to be able to
+use storybook with the latest tailwind version.
+
+## Project setup
+
+This recipe has to be run from the root folder of a mono-repository created with
+`amazee-recipes create-monorepo`.
+
+```typescript
+// Pick project name from package.json.
+const { name: projectName } = $$.file('package.json');
+```
+
+Create a folders structure to contain the UI
+
+```typescript
+$$(`mkdir -p packages/${projectName}/ui`);
+$$.chdir(`packages/${projectName}/ui`);
+```
+
+Initialize the Ui project folder with yarn
+
+```typescript
+// yarn init
+$$('yarn init -y -p');
+```
+
+Add the project namespace
+
+```typescript
+// add the project namespace
+$$.file('package.json', (json) => ({
+  ...json,
+  name: `${projectName}-ui`,
```

Comment:
I've intentionally used '-' instead of '/' otherwise it will break recipes build cause the name is not valid for the regex rule.
Testing it inside '@amazeelabs/recipes' so the result would be '@amazeelabs/recipes/ui'